jest.config.mjs

```diff
@@ -0,0 +1,31 @@
+import nextJest from 'next/jest.js';
+
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
```

Comment:
I think we can remove these unnecessary comments.

Reply:
will do